src/engine/UIManager.cxx

```diff
@@ -160,16 +160,26 @@ void UIManager::init()
           dynamic_cast<Button *> (uiElement.get())->isToggleButton = toggleButton;
           dynamic_cast<Button *> (uiElement.get())->drawImageButtonFrame(drawFrame);
           break;
-        case ElementType::TextButton:
-          uiElement = std::make_unique<Button>(elementRect);
-          dynamic_cast<Button *>(uiElement.get())->setText(text);
-          dynamic_cast<Button *>(uiElement.get())->isToggleButton = toggleButton;
+        case ElementType::TextButton: {
+          int fontSize = element.value("FontSize", 0);
```

Comment:
Why 0? Use a default value set in settings.json when there is no parameter is set in UiLayout.json